opsdroid/connector/telegram/__init__.py

```diff
@@ -164,7 +164,10 @@ def build_url(self, method):
             elif "message" in result and "text" in result["message"]:
                 user = self.get_user(result)
                 message = Message(
-                    result["message"]["text"], user, result["message"]["chat"], self
+                    text=result["message"]["text"],
+                    user=user,
```

Comment:
I am 90% sure telegram has user IDs, so we would want to add them to this as well.

Reply:
You might have to update the `self.get_user` helper method but yeah telegram does return user ids as well.

This is what Telegram returns:
```python
{'ok': True, 'result': [{'update_id': 664568166, 'message': {'message_id': 1198, 'from': {'id': 639889348, 'is_bot': False, 'first_name': 'Fabio', 'last_name': 'Rosado', 'username': 'FabioRosado', 'language_code': 'en'}, 'chat': {'id': 639889348, 'first_name': 'Fabio', 'last_name': 'Rosado', 'username': 'FabioRosado', 'type': 'private'}, 'date': 1571692254, 'text': 'hi'}}]}
```

So you can find the user id at `response["message"]["from"]["id"]`

Reply:
I changes `self.get_user`  now it returns `user` as well as `user_id`